Swiften/Elements/MIXCreate.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2017 Tarun Gupta
+ * Licensed under the simplified BSD license.
+ * See Documentation/Licenses/BSD-simplified.txt for more information.
+ */
+
+#pragma once
+
+#include <memory>
+
+#include <boost/optional.hpp>
+
+#include <Swiften/Base/API.h>
+#include <Swiften/Base/Override.h>
```

Comment:
You include `Override.h` but don't use it. Feel free to use it, by appending `SWIFTEN_OVERRIDE` to method declarations you override.  Although I think here is no method you currently override.